test/extended/testdata/bindata.go

```diff
@@ -12149,7 +12151,8 @@ var _examplesDbTemplatesMongodbEphemeralTemplateJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${MONGODB_USER}\n       Password: ${MONGODB_PASSWORD}\n  Database Name: ${MONGODB_DATABASE}\n Connection URL: mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/blob/master/3.2/README.md.",
   "labels": {
-    "template": "mongodb-ephemeral-template"
+    "template": "mongodb-ephemeral-template",
+    "app": "mongodb-ephemeral-template"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.